packages/app/src/model2/orbitStoreFactory.ts

```diff
@@ -6,3 +6,7 @@ export async function createOrbitStore(
 ): Promise<OrbitStore> {
   return new OrbitStoreFS(databaseName);
 }
+
+export async function createDefaultOrbitStore(): Promise<OrbitStore> {
```

Comment:
moved this to be a helper function so that the ingester handler can init the store outside the React lifeycle.